src/diffpy/labpdfproc/functions.py

```diff
@@ -172,28 +172,25 @@ def get_path_length(self, grid_point, angle):
         return total_distance, primary_distance, secondary_distance
 
 
-def compute_cve(diffraction_data, mud, wavelength):
+def compute_cve(mud, wavelength):
     """
-    compute the cve for given diffraction data, mud and wavelength
+    compute the cve for given mud
 
     Parameters
     ----------
-    diffraction_data Diffraction_object
-      the diffraction pattern
     mud float
       the mu*D of the diffraction object, where D is the diameter of the circle
     wavelength float
       the wavelength of the diffraction object
 
     Returns
     -------
-    the diffraction object with cve curves
+    the diffraction object with cve curves on the resampled grid
 
     it is computed as follows:
-    We first resample data and absorption correction to a more reasonable grid,
-    then calculate corresponding cve for the given mud in the resample grid
-    (since the same mu*D yields the same cve, we can assume that D/2=1, so mu=mud/2),
-    and finally interpolate cve to the original grid in diffraction_data.
+    We resample data and absorption correction to a more reasonable grid,
```

Comment:
This is a very excellent explanation, but normally something like this that expands on what the function does, I think it is place below the first line of the docstring after a blank line, not down here at the bottom.